src/clients/inst_sap-start.rb

```diff
@@ -67,7 +76,7 @@ def main
       ret
     end
 
-    def constumize_sap_installation(start_wizard, start_rdp)
+    def costumize_sap_installation(start_wizard, start_rdp)
```

Comment:
still I think it is typo and should be "customize" not?

Reply:
Oooops it is a big typo :-)

Reply:
I've fixed it